### ISSUE TITLE
how to add a clone of typical ETF? 

### ISSUE THREAD

**Issue by tomazb** [jamsix/ib-edavki#126]
Hello!

I used Milano version of SDIV (see [https://finance.yahoo.com/quote/SDIV.MI]). What would be the company data for this?

**Comment by tomazb** [comment]
`<company>
        <conid>544928174</conid>
        <symbol>SDIV</symbol>
        <name></name>
        <taxNumber></taxNumber>
        <address></address>
        <country>IT</country>
    </company>`

**Comment by tomazb** [comment]
and

XLU,  XLE as contracts for difference

   <company>
        <conid>134770389</conid>
        <symbol>XLU</symbol>
        <name></name>
        <taxNumber></taxNumber>
        <address></address>
        <country></country>
    </company>
    <company>
        <conid>134770358</conid>
        <symbol>XLE</symbol>
        <name></name>
        <taxNumber></taxNumber>
        <address></address>
        <country></country>
    </company>

**Comment by pronebird** [comment]
AFAIK you don't need to update `companies.xml` with data for CFDs unless you receive dividends which I believe are passed through for CFDs. Typically I'd simply scan my XML output from IBKR to find the contract id (conid) by symbol. But you can also find it online via IBKR: https://pennies.interactivebrokers.com/cstools/contract_info/v3.10/index.php

Company address and tax number are manual labour, usually for the US securities SEC filings contain "employer id", for other securities I'd google it or contact IR.

**Comment by tomazb** [comment]
Still, getting info for OXY was easy. But getting them for ETFs and CFDs is tricky. I can do a PR for OXY.

**Comment by tomazb** [comment]
Info for OXY is here - https://eintaxid.com/company/954035997-occidental-petroleum-corp-%2Fde%2F/

**Comment by pronebird** [comment]
Yeah it's a known problem, I have some companies in portfolio that have origin in UK but absolutely zero tax info.

**Comment by jamsix** [comment]
Regarding CFDs - CFD is a contract between you and IBKR, hence I _assume_ the dividend payer is IBKR, not the company of the underlying asset (stock)?

**Comment by tomazb** [comment]
The real payer would be the company behind it, as with any stock.

```
<company>
        <conid>134770389</conid>
        <symbol>XLU</symbol>
        <name>Utilities Select Sector SPDR Fund (XLU@ARCA)</name>
        <taxNumber>20-3783731</taxNumber>
        <address>11 Wall Street, New York, NY 10005</address>
        <country>US</country>
    </company>
```

>

**Comment by tomazb** [comment]
I have the data for XLE and XLU
```
   <company>
        <conid>134770389</conid>
        <symbol>XLU</symbol>
        <name>Utilities Select Sector SPDR Fund (XLU@ARCA)</name>
        <taxNumber>20-3783731</taxNumber>
        <address>11 Wall Street, New York, NY 10005</address>
        <country>US</country>
    </company>
    <company>
        <conid>134770358</conid>
        <symbol>XLE</symbol>
        <name>Energy Select Sector SPDR Fund (XLE@ARCA)</name>
        <taxNumber>20-3783731</taxNumber>
        <address>11 Wall Street, New York, NY 10005</address>
        <country>US</country>
    </company>
```

and missing for Global X Superdividend UCITS ETF (SDIV@EBS)

```
<company>
        <conid>544928174</conid>
        <symbol>SDIV</symbol>
        <name>Global X Superdividend UCITS ETF (SDIV@EBS)</name>
        <taxNumber></taxNumber>
        <address>2nd Floor, Block E, Iveagh Court, Harcourt Road, Dublin 2, Ireland, D02 T330</address>
        <country>IE</country>
    </company>
```

**Comment by jamsix** [comment]
> The real payer would be the company behind it, as with any stock.

Yes, but from your perspective, you have a contract with IBKR, and they pay out the dividend. You do not own the original company stock and are not eligible to receive payment from the original company. Moght be a question for FURS.
